### PR TITLE
Heroku setup and some fixes

### LINKED ISSUE #48: Setup for Heroku
## Description

Set everything up to run properly on Heroku.

## Things to do

* Setup redis
* Add dyno for Sidekiq

### PR DESCRIPTION
**Pull request by hdamico**
## What does this PR do?

Setups everything to run properly on Heroku and fixes some tests and code (just a little)

Resolves #48

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -20,5 +20,7 @@ class Application < Rails::Application
     config.active_record.schema_format = :sql
     # Autoload lib folder
     config.autoload_paths << "#{Rails.root}/lib"
+
+    config.hosts << 'code-review-metrics.herokuapp.com'
```
Comment (santiagovidal) [inline config/application.rb]: This should be an Env variable.

File: Procfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+web: bin/rails server -p $PORT -e $RAILS_ENV
```
Comment (juanmanuelramallo) [inline Procfile]: This should actually use puma

```suggestion
web: bundle exec puma -C config/puma.rb
```

https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server#procfile

File: .env.example [hunk 3]
```diff
@@ -4,3 +4,5 @@ GITHUB_ANALYZER_PASSWORD=postgres
 
 # Must be set as the webhook secret on your github app
 GITHUB_ANALYZER_WEBHOOK_SECRET=fake-secret
+REDIS_URL=heroku_redis_url
+SIDEKIQ_THREADS=3
```
Comment (horacio) [inline .env.example]: Aren't we setting the concurrency twice? Here and then in the Procfile? 👀

Comment (hdamico) [reply .env.example]: Not really, the worker dyno actually is not working, i should delete that dyno.

**Pull request merged**